meinberlin/apps/plans/forms.py

```diff
@@ -49,6 +50,7 @@ def __init__(self, *args, **kwargs):
         super().__init__(*args, **kwargs)
         self.fields['district'].empty_label = _('City wide')
         self.fields['contact_address_text'].widget.attrs['rows'] = 6
+        self.fields['participation_explanation'].widget.attrs['rows'] = 1
```

Comment:
as I understood, it should look like address field, so also 6 rows here?

Reply:
address field reference was just to ensure linebreaks so it should actually go in the user side story, in designs it is just one line but can be expanded